.github/workflows/ci.yaml

```
@@ -35,8 +35,24 @@ jobs:
           cd azure_storage
           tox run -e lint
 
-  unit-test:
-    name: Unit tests
+  lint-s3:
+    name: Lint s3-integrator
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
```

nitpick: `actions/checkout` has recently been updated to [v5](https://github.com/actions/checkout/releases/tag/v5.0.0) that uses `node.js` `24`, we could update here
```suggestion
        uses: actions/checkout@v5
```